RetailCoder.VBE/App.cs

```diff
@@ -169,13 +170,29 @@ private void _configService_SettingsChanged(object sender, ConfigurationChangedE
             }
         }
 
+        private void EnsureDirectoriesExist()
+        {
+            try
+            {
+                if (!Directory.Exists(ApplicationConstants.LOG_FOLDER_PATH))
+                {
+                    Directory.CreateDirectory(ApplicationConstants.LOG_FOLDER_PATH);
+                }
+            }
+            catch
+            {
+                //Does this need to display some sort of dialog?
```

Comment:
I'm not even sure we should catch here. I daresay a Fail-Fast is better. Blow up rather sooner than later and either let this bubble or show a dialog and stop loading RD.  
Since we would blow up sooner or later anyways with the directory not existing or having failed to be created we can basically stop the world (at least the RD world)